public/api-docs/swagger_doc.yaml

```diff
@@ -0,0 +1,193 @@
+                        type: string
+                        example: Conditional permission
+                      decision_issued_at:
+                        type: string
+                        example: '2022-10-14'
+                      validated_at:
+                        type: string
+                        example: '2022-06-23'
+                      view_documents:
+                        type: string
+                        example: 'https://example.gov.uk'
+                      uprn:
+                        type: string
+                        example: '123456789123'
+                      code:
+                        type: string
+                        example: RD02
+                      type:
+                        type: string
+                        example: Residential
+                      full:
+                        type: string
+                        example: '123 High Street, Big City, AB3 4EF'
+                      town:
+                        type: string
+                        example: Big City
+                      postcode:
+                        type: string
+                        example: AB3 4EF
+                      map_east:
+                        type: string
+                        example: '123456'
+                      map_north:
+                        type: string
+                        example: '654321'
+                      latitude:
+                        type: string
+                        example: '55.703226'
+                      longitude:
+                        type: string
+                        example: '-6.403439'
+                      ward_code:
+                        type: string
+                        example: W16
+                      ward_name:
+                        type: string
+                        example: Ward 16
+            example:
+              planning_applications:
+                - reference: AB/22/1234/FA
+                  application_type_code: FA
+                  application_type: Full Application
+                  area: South
+                  description: Demolition of existing first floor.
+                  received_at: '2022-05-30'
+                  assessor: Keira Jones
+                  reviewer: Leila Smith
+                  decision: Conditional permission
+                  decision_issued_at: '2022-10-14'
+                  validated_at: '2022-06-23'
+                  view_documents: 'https://example.gov.uk'
+                  uprn: '123456789123'
+                  code: RD02
+                  type: Residential
+                  full: '123 High Street, Big City, AB3 4EF'
+                  town: Big City
+                  postcode: AB3 4EF
+                  map_east: '123456'
+                  map_north: '654321'
+                  latitude: '55.703226'
+                  longitude: '-6.403439'
+                  ward_code: W16
+                  ward_name: Ward 16
```

Comment:
What's the difference with [this](https://github.com/unboxed/paapi/blob/main/public/api-docs/v1/swagger_doc.yaml)?